docs-archive/apache-airflow-providers-apache-drill/1.0.0/index.html

```diff
@@ -330,7 +330,6 @@
 </ul>
 <p class="caption"><span class="caption-text">Resources</span></p>
 <ul>
-<li class="toctree-l1"><a class="reference external" href="https://github.com/apache/airflow/tree/main/airflow/providers/apache/drill/example_dags">Example DAGs</a></li>
```

Comment:
All of the references to Apache Drill DAGs disappear?

Reply:
Yes. It never had any examples :). So the links were just wrong.

Reply:
But it does for that release: https://github.com/apache/airflow/tree/providers-apache-drill/1.0.0/airflow/providers/apache/drill/example_dags

Granted it didn't have an `__init__.py` file in that directory for 1.0.0, but I assume that doesn't matter?

Reply:
Same pattern with the newest provider, ArangoDB: https://github.com/apache/airflow/tree/providers-arangodb/1.0.0/airflow/providers/arangodb/example_dags